conf/messages

```diff
@@ -772,6 +772,13 @@ benefits.entertainmentBenefitAmount.error.overMaximum.agent = Your client’s en
 benefits.entertainmentBenefitAmount.error.invalidFormat.individual = Enter the amount you got for entertainment in the correct format
 benefits.entertainmentBenefitAmount.error.invalidFormat.agent = Enter the amount your client got for entertainment in the correct format
 
+benefits.telephoneBenefit.title.individual = Did you get a benefit for using a telephone?
```

Comment:
It seams there is inconsistency in the properties keys. Sometimes we use Benefit, sometimes Benefits. Do you feel we need to change it?